src/helper/proxy/index.ts

```diff
@@ -45,6 +68,7 @@ const buildRequestInitFromRequest = (
     method: request.method,
     body: request.body,
     duplex: request.body ? 'half' : undefined,
+    ...requestInit,
     headers,
```

Comment:
Looking at this again, maybe this should actually be..

```suggestion
    ...requestInit,
    method: request.method,
    body: request.body,
    duplex: request.body ? 'half' : undefined,
    headers,
```

..to avoid `requestInit` overwriting any of `method`, `body`, `duplex` and `headers`?